### ISSUE TITLE
Improve rendering of images

### ISSUE THREAD

**Issue by yoshuawuyts** [Jam3/math-as-code#2]
It appears many notations can also be expressed as unicode. Not sure if it's useful, but it might be worth considering. So far these are the pros / cons I could think of:

__pro__
- sharable! It's just plain text so it'll work anywhere.
- crisper rendering, though saving mathjax output as svg might achieve the same

__con__
- it's visually different from formulas out in the wild (e.g. LaTeX rendering)

## Example
```
100
 ∑ i
i=1
```
```
100
 ∑ (2i+1)
i=1
```
```
 3   6
 ∑   ∑ (3ij)
i=1 j=4
```
```
 6
 Π i
i=1
```
```
â
```

Thanks!

**Comment by mattdesl** [comment]
I think in some cases it will make sense, like describing `â`. 

However the end goal is to demystify the equations in graphics papers, so I think it's best just to stick to images. 

SVG would be best, maybe it could be automated cleanly.

**Comment by yoshuawuyts** [comment]
Ah yeah, that makes sense. Renaming this issue.

**Comment by StoneCypher** [comment]
I strongly disagree.  Unicode can be carried (eg by cut and paste) to places that images cannot, renders well on a much broader variety of devices, etc.

Using images to replace unicode is an HTML3 era approach.  Please reconsider.

**Comment by allo-** [comment]
There is MathJAX, another option is to render LaTeX into svg (i.e. via eps as intermediate format) or LaTeX into png. Or just create a PDF instead of a html (markdown) document.

Trying to use unicode will fail for many interesting formulas.

**Comment by mattdesl** [comment]
All images are hotlinking to codecogs and using SVG format. For now this ticket is fixed, but it has introduced a new problem: #29.

Closing..
